webcomponents/tool/src/main/frontend/package.json

```diff
@@ -24,7 +24,6 @@
     "@popperjs/core": "2.9.3",
     "@webcomponents/webcomponentsjs": "2.6.0",
     "cropperjs": "1.5.12",
-    "flatpickr": "4.6.9",
```

Comment:
Do you know if the flatpickr webjar can be removed from the library project?

Reply:
that's a good question. This was just using it from node_modules, so is unrelated.

Reply:
Actually, I see what you mean. Assignments brings it in. Maybe that was a leftover from before I moved to the npm module.